### ISSUE TITLE
First node in fresh cluster loses his attrs

### ISSUE THREAD

**Issue by tschottdorf** [cockroachdb/cockroach#1006]
> rm -rf /tmp/node1; ./cockroach init --stores=ssd=/tmp/node1 && ./cockroach start --scan-interval=1s --insecure --stores=ssd=/tmp/node1 --addr=:8080 --gossip=self:// --v=1 2>&1 | tee 1.log

```
$ ./cockroach range ls --insecure
""-"\xff\xff" [1]
	0: node-id=1 store-id=1 attrs=[]
```

**Comment by tschottdorf** [comment]
replicas which get the range have the `ssd` attr set though.

**Comment by spencerkimball** [comment]
Actually, let's change this issue a bit. What we want to do is never store attributes in `RangeDescriptor.[]Replica` because they can change. Instead, we should only ever gossip them, which means that dist_sender needs to look up attributes via gossip (it's looking up the address of a node anyway).

**Comment by tamird** [comment]
Fixed in #1212.
